tests/src/runtimeApi/memory/hipMemcpy.cpp

```diff
@@ -292,8 +292,8 @@ void memcpytest2_get_host_memory(size_t& free, size_t& total) {
     MEMORYSTATUSEX status;
     status.dwLength = sizeof(status);
     GlobalMemoryStatusEx(&status);
-    free = status.ullAvailPhys;
-    total = status.ullTotalPhys;
+    free = (0.4 * status.ullAvailPhys);
+    total = (0.4 * status.ullTotalPhys);
 }
```

Comment:
Please, add an explicit typecast and put in a comment a few words explaining the chosen heuristic number 0.4.